doc/reference/reference_lua/datetime.rst

```diff
@@ -263,9 +263,14 @@ Functions
     *   RFC 3339
     *   extended `strftime <https://www.freebsd.org/cgi/man.cgi?query=strftime&sektion=3>`__ -- see description of the :ref:`format() <datetime-format>` for details.
 
+    By default fields that are not specified are equal to appropriate values in a Unix time.
+
+    Leap second is supported, see a section :ref:`leap second <leap-second>`.
```

Comment:
Link to section `Leap second` doesn't work. Andrey, could you help please? @andreyaksenov

Reply:
Fixed.